share/picongpu/tests/CurrentDeposition/lib/python/test/CurrentDeposition/grid_class.py

```diff
@@ -0,0 +1,194 @@
+import numpy as np

```

Comment:
This file misses a general deocumentation at the beginning explaining its purpose. (just few lines)

Reply:
done